site/src/pages/WorkspacesPage/WorkspacesEmpty.tsx

```diff
@@ -83,70 +67,30 @@ export const WorkspacesEmpty: FC<WorkspacesEmptyProps> = ({
 			description={`${defaultMessage} Select one template below to start.`}
 			cta={
 				<div>
-					<div
-						css={{
-							display: "flex",
-							flexWrap: "wrap",
-							gap: 16,
-							marginBottom: 24,
-							justifyContent: "center",
-							maxWidth: "800px",
-						}}
-					>
+					<div className="flex flex-wrap gap-4 mb-6 justify-center max-w-[800px]">
 						{featuredTemplates?.map((t) => (
 							<Link
 								key={t.id}
 								to={`${getLink(
 									linkToTemplate(t.organization_name, t.name),
 								)}/workspace`}
-								css={(theme) => ({
-									width: "320px",
-									padding: 16,
-									borderRadius: 6,
-									border: `1px solid ${theme.palette.divider}`,
-									textAlign: "left",
-									display: "flex",
-									gap: 16,
-									textDecoration: "none",
-									color: "inherit",
-
-									"&:hover": {
-										backgroundColor: theme.palette.background.paper,
-									},
-								})}
+								className="w-[320px] p-4 rounded-md border border-divider text-left flex gap-4 no-underline text-inherit hover:bg-paper"
```

Comment:
the old `theme.palette.divider` and `theme.palette.background.paper` colors haven't been carried over from material ui to tailwind, so `border-divider` and `hover:bg-paper` are not actually valid classes.

if you look at the failing chromatic tests, you can see what it looked like before and what it looks like now. you can see the set of available tailwind colors in our tailwind.config.js file, and their actual values in index.css. the match doesn't have to be exact, but the colors should be close.